index.html

```diff
@@ -83,6 +83,204 @@ <h1>
+                  </ul>
+                </div>
+                <p class="d-text">
+                  A daily selection of privately personalized reads; no accounts
+                  or sign-ups required.
+                </p>
+                <ul class="lang">
+                  <li class="lang-item">html</li>
+                  <li class="lang-item">css</li>
+                  <li class="lang-item1">javascript</li>
+                </ul>
+                <button class="project-button">See Project</button>
+              </div>
+            </div>
+        </div>
+
+
+        <div class="main-container">
+            <div class="introduction">
+              <h1>About Myself</h1>
+              <p>
+                I’m a software developer! I can help you build a product, feature
+                or website Look through some of my work and experience! If you
+                like what you see and have a project you need coded, don’t
+                hestiate to contact me.
+              </p>
+            </div>
+            <div class="connect">
+              <a href="#">LET’S CONNECT</a>
+              <div class="intro-card2">
+                <ul class="icon">
+                  <li>
+                    <a href="#"> <img src="img/Vector.svg" /> </a>
+                  </li>
+                  <li>
+                    <a href="#">
+                      <img
+                        src="img/Vector (1).svg"
+                      />
+                    </a>
+                  </li>
+                  <li>
+                    <a href="#"> <img src="img/Linkedin icon.svg" /> </a>
+                  </li>
+                  <li>
+                    <a href="#"> <img src="img/Vector (3).svg" /> </a>
+                  </li>
+                  <li>
+                    <a href="#"> <img src="img/Vector (4).svg" /> </a>
+                  </li>
+                </ul>
+               </ul>
+                 <button class="project-button">Get my resume</button>
+              </div>
+              </div>
+            </div>
+        </div>
+
+        <div class="icon2">
+            <ul class="lang-wrap">
+                <li class="lang-icon">
+                    <h4>Languages</h4>
+                    <svg width="24" height="24" viewBox="0 0 24 24" fill="none" xmlns="http://www.w3.org/2000/svg">
+                        <path fill-rule="evenodd" clip-rule="evenodd" d="M5.29289 8.29289C5.68342 7.90237 6.31658 7.90237 6.70711 8.29289L12 13.5858L17.2929 8.29289C17.6834 7.90237 18.3166 7.90237 18.7071 8.29289C19.0976 8.68342 19.0976 9.31658 18.7071 9.70711L12.7071 15.7071C12.3166 16.0976 11.6834 16.0976 11.2929 15.7071L5.29289 9.70711C4.90237 9.31658 4.90237 8.68342 5.29289 8.29289Z" fill="#5E6C84"/>
+                    </svg>
+                </li>
+
+            </ul>
+            <ul class="languages">
+                <li class="languages_details">
+                    <img src="img/IMG 5.png" alt="javascript logo">
```

Comment:
- [ ] In your languages section, kindly ensure that the language logos are perfect circles to match the original design
- [ ] You can also reduce the padding in the `skills` and `frameworks` sections to match the original design

![image](https://user-images.githubusercontent.com/98453979/219645973-a445801f-23b7-4e26-aa4e-1d7cfe7490e3.png)

### Checkout the original design
![image](https://user-images.githubusercontent.com/98453979/219646080-5bee0e9c-5a8c-40f1-af4c-955857f5d525.png)